admin/build-release.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Script that builds a GMT release and makes the compressed tarballs and macOS Bundle
+Version=6.0.0
+if [ $# -eq 0 ]; then
+	echo "Usage: build-release.sh tag"
+	echo "e.g., build-release.sh rc3"
+	echo "If no tag (i.e., a final 6.x.y version) then give - as tag"
+	echo "build-release.sh must be run from top-level gmt directory"
+	exit 1
+fi
+if [ ! -d cmake ]; then
+	echo "Must be run from top-level gmt directory"
+	exit 1
+fi
+if [ ! "X$1" = "X-" ]; then
+	tag=$1
+fi
+
+# 1. Set basic ConfigUser.cmake file for a release build
+if [ -f cmake/ConfigUser.cmake ]; then
+	cp cmake/ConfigUser.cmake cmake/ConfigUser.cmake.orig
+fi
+cp -f admin/ConfigReleaseBuild.cmake cmake/ConfigUser.cmake
+# 2. Make build dir and configure it
+rm -rf build
+mkdir build
+cd build
+cmake -G Ninja -DGMT_PACKAGE_VERSION_SUFFIX=${tag} -DGMT_PUBLIC_RELEASE=TRUE ..
```

Comment:
We need set GMT_PACKAGE_VERSION_SUFFIX and GMT_PUBLIC_RELEASE in the `cmake/ConfigDefault.cmake` file and commit the changes before making a release. So we don't need set them in the command line.

Reply:
OK, will remove those too.